exporter/opensearchexporter/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenSearch Exporter
+
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [devel]   |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
+
+This exporter supports sending OpenTelemetry signals as documents to [OpenSearch](https://www.opensearch.org).
+
+The documents are sent using [observability catalog](https://github.com/opensearch-project/opensearch-catalog/tree/main/schema/observability) schema.
+
+## Configuration options
+
+- `endpoints`: List of OpenSearch URLs. If endpoints is missing, the
+  OPENSEARCH_URL environment variable will be used.
+- `num_workers` (optional): Number of workers publishing bulk requests concurrently.
+- `dataset` (default=`default`) a user-provided label. It is used to construct the name of the destination index or data stream.
+- `namespace` (default=`namespace`)a user-provided label. It is used to construct the name of the destination index or data stream.
+- `flush`: Event bulk buffer flush settings
+    - `bytes` (default=5242880): Write buffer flush limit.
+    - `interval` (default=30s): Write buffer time limit.
```

Comment:
We [generally recommend](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#general-recommendations) against introducing any batching within an exporter.  This is better handled via existing processors that can handle batching telemetry data.

Reply:
These are settings internal to the OpenSearch client. 

The exporter may create several JSON documents for one `pdata.Traces` record.  These settings control how these documents are sent to OpenSearch. The exporter does not do any batching of records it receives from the collector.

Reply:
Why the `interval` value, then?  I can understand having a maximum export data size requiring splitting an incoming record into multiple outgoing requests, but the `interval` value implies to me that the exporter will hold onto incoming records for some period of time before sending them if the maximum data size is not yet reached.

Reply:
Yes, current exporter implementation maintains one bulk indexer object from opensearch-go client which, in turn maintains this queue.

I believe it's reasonable to create bulk indexer for each `pushTraceData` call and make this setting unnecessary.